HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -272,5 +365,6 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
 
     process = customizeHLTfor42497(process)
     process = customizeHLTfor42943(process)
+    process = customizeHLTfor43257(process)
```

Comment:
Sorry, but so far alpka PRs were not touching the HT, just providing code and infrastructure for later use/migration. What has changed here?

Reply:
I do not know for sure but maybe, unlike this one, the previous alpaka PRs did not touch anything that is running at the HLT.
The other ECAL alpaka PR for example only added new data formats.

Reply:
Without the customization the CUDA version should run at the HLT while the alpaka code would run for the reconstruction.

Reply:
For me HLT staying with CUDA (as already in use) for now is fine, until we have a 'HLT migration to alpka' plan. I have no opinion on what reconstruction should or should not do.

Reply:
> the previous alpaka PRs did not touch anything that is running at the HLT

Yes, this is the first PR that could be used at HLT out of the box... and we have not discussed how to integrate the alpaka changes at HLT.

Maybe something for the TSG coordination next week ?

Reply:
Yes!

Reply:
Suggestion: maybe call this specific alpka function from a generic alpaka function (similar to ```def customiseFor2018Input(process)``` also to be defined in this file where this generic alpaka migration function itself is currently not called and thus only used when the expert running a test explicitly specifies for it in the cmsDriver command. This allows to collect all the alpaka bits and makes them available for expert testing now, without disturbing the standard workflows.

Reply:
Defined the customisation function but not calling it now.